client/client.go

```diff
@@ -1861,58 +1861,132 @@ func (c *Client) emitStats() {
 	}
 }
 
-// emitHostStats pushes host resource usage stats to remote metrics collection sinks
-func (c *Client) emitHostStats(hStats *stats.HostStats) {
+func (c *Client) setGaugeForMemoryStats() {
 	nodeID := c.Node().ID
-	metrics.SetGauge([]string{"client", "host", "memory", nodeID, "total"}, float32(hStats.Memory.Total))
-	metrics.SetGauge([]string{"client", "host", "memory", nodeID, "available"}, float32(hStats.Memory.Available))
-	metrics.SetGauge([]string{"client", "host", "memory", nodeID, "used"}, float32(hStats.Memory.Used))
-	metrics.SetGauge([]string{"client", "host", "memory", nodeID, "free"}, float32(hStats.Memory.Free))
+	hStats := c.hostStatsCollector.Stats()
 
-	metrics.SetGauge([]string{"uptime"}, float32(hStats.Uptime))
+	if !c.config.DisableTaggedMetrics {
+		labels := []metrics.Label{{"node_id", nodeID}, {"datacenter", c.Node().Datacenter}}
```

Comment:
Why don't the others have datacenter?

Reply:
Create a base labels on the client as well and where addition labels are needed they can just append and create a new slice?